tests/test_overfocus.py

```diff
@@ -179,8 +183,8 @@ def test_detector_specimen_px(params):
 def test_project():
     size = 16
     params = OverfocusParams(
-        overfocus=0.1,
-        scan_pixel_size=0.1,
+        overfocus=1,
+        scan_pixel_size=0.5,
```

Comment:
It is easy to test only if there's an integer relation between object pixels and result pixels so that the object can be extracted with NumPy slicing. I've added more test cases to make it clearer.

Reply:
Ah, makes sense!